api/app/src/external/commonwell/api.ts

```diff
@@ -0,0 +1,74 @@
+import { APIMode, CommonWell, PurposeOfUse, RequestMetadata } from "@metriport/commonwell-sdk";
+import { Config, getEnvVarOrFail } from "../../shared/config";
+
+// TODO move this to Config
+const metriportOrgName = getEnvVarOrFail("CW_ORG_NAME");
+const metriportSandboxOrgName = getEnvVarOrFail("SANDBOX_CW_ORG_NAME");
+
+const metriportSandboxOID = getEnvVarOrFail("SANDBOX_SYSTEM_ROOT_OID");
+const metriportOID = getEnvVarOrFail("SYSTEM_ROOT_OID");
+const metriportPrivateKey = getEnvVarOrFail("CW_PRIVATE_KEY");
+const metriportCert = getEnvVarOrFail("CW_CERTIFICATE");
+
+const memberManagementPrivateKey = getEnvVarOrFail("CW_MEMBER_PRIVATE_KEY");
+const memberManagementCert = getEnvVarOrFail("CW_MEMBER_CERTIFICATE");
+const memberManagementOID = getEnvVarOrFail("CW_MEMBER_OID");
+
+const apiMode = Config.isProdEnv() ? APIMode.production : APIMode.integration;
+export const apiUrl = Config.isProdEnv()
+  ? Config.getCWProductionUrl()
+  : Config.getCWProductionUrl();
+
+export const CW_ID_PREFIX = "urn:oid:";
+export const CW_ID_URL_ENCODED_PREFIX = `%5E%5E%5Eurn%3aoid%3a`;
+
+export const commonWell = new CommonWell(
+  metriportCert,
+  metriportPrivateKey,
+  metriportOrgName,
+  metriportOID,
+  apiMode
+);
+export const commonWellSandbox = new CommonWell(
+  metriportCert,
+  metriportPrivateKey,
+  metriportSandboxOrgName,
+  metriportSandboxOID,
+  APIMode.integration
+);
```

Comment:
Some things to consider:
- We shouldn't need `metriportSandboxOrgName` + `metriportSandboxOID` -> can just configure the env vars appropriately for each env
- Then, we won't need a separate `commonWellSandbox`

A point that will be up for debate is whether we should have our sandbox env talking to CW, or only our staging env. It seems that opening up our integration CW env to sandbox users could be a mistake, since that's the only testing environment we will have, and it can quickly become polluted with junk data that we can never delete (ie you can't delete an org from CW once it's created).

@Orta21 @leite08 -> thoughts on the last point?

Reply:
I actually meant to remove the sandbox stuff, i just hadnt pushed up the commit for that change. Should be removed now